<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>graphwindow</class>
 <widget class="QDialog" name="graphwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="window_name">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>551</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 225, 200);
	color:rgb(87, 87, 87)
 }</string>
   </property>
   <property name="text">
    <string>Окно построения графиков</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLineEdit" name="function">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>19</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 225, 200);
	color:rgb(87, 87, 87)
 }</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="window_name_2">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 190, 173);
	color:rgb(87, 87, 87)
 }
</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Введите функцию график&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;которой вы хотите построить&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>140</y>
     <width>271</width>
     <height>83</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0">
     <widget class="QLabel" name="window_name_5">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 190, 173);
	color:rgb(87, 87, 87)
 }
</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;X-max&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="window_name_3">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 190, 173);
	color:rgb(87, 87, 87)
 }
</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Введите значения Х&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="window_name_4">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 190, 173);
	color:rgb(87, 87, 87)
 }
</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;X-min&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="x_min">
      <property name="styleSheet">
       <string notr="true">QDoubleSpinBox {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 225, 200);
	color:rgb(87, 87, 87)
 }</string>
      </property>
      <property name="minimum">
       <double>-1000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="x_max">
      <property name="styleSheet">
       <string notr="true">QDoubleSpinBox {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 225, 200);
	color:rgb(87, 87, 87)
 }</string>
      </property>
      <property name="minimum">
       <double>-1000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>240</y>
     <width>271</width>
     <height>83</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="2" column="0">
     <widget class="QLabel" name="window_name_7">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 190, 173);
	color:rgb(87, 87, 87)
 }
</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Y-max&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="y_min">
      <property name="styleSheet">
       <string notr="true">QDoubleSpinBox {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 225, 200);
	color:rgb(87, 87, 87)
 }</string>
      </property>
      <property name="minimum">
       <double>-1000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="y_max">
      <property name="styleSheet">
       <string notr="true">QDoubleSpinBox {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 225, 200);
	color:rgb(87, 87, 87)
 }</string>
      </property>
      <property name="minimum">
       <double>-1000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="window_name_8">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 190, 173);
	color:rgb(87, 87, 87)
 }
</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Y-min&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="window_name_6">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 190, 173);
	color:rgb(87, 87, 87)
 }
</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Введите значения Y&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>390</y>
     <width>271</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_create_graph">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>19</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 225, 200);
	color:rgb(87, 87, 87)
 }

 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }</string>
      </property>
      <property name="text">
       <string>Построить график</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_clear_graph">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>19</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 225, 200);
	color:rgb(87, 87, 87)
 }

 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }</string>
      </property>
      <property name="text">
       <string>Очистить данные</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>540</y>
     <width>551</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton_back_to_calc">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 190, 173);
	color:rgb(87, 87, 87)
 }

 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }</string>
      </property>
      <property name="text">
       <string>Вернуться в калькулятор</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCustomPlot" name="graphic" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>551</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4"/>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>330</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="window_name_9">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 190, 173);
	color:rgb(87, 87, 87)
 }
</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Установите шаг&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="step">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QDoubleSpinBox {
     border: 1px solid #8f8f91;
     background-color:rgb(235, 225, 200);
	color:rgb(87, 87, 87)
 }</string>
      </property>
      <property name="minimum">
       <double>0.050000000000000</double>
      </property>
      <property name="maximum">
       <double>1.500000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>library/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
